### ISSUE TITLE
Crash when right click copying text

### ISSUE THREAD

**Issue by drichardson** [microsoft/terminal#2881]
Windows Terminal Preview crashes when using right click to copy multiple lines of text in PowerShell 7.0.0-preview.4.

[Feedback Hub link with crash](https://aka.ms/AA6486q)

# Environment

```none
Windows build number: 10.0.18362.0
Windows Terminal version (if applicable): 0.5.2661.0

Any other software?

PowerShell 7.0.0-preview4

```

# Steps to reproduce

1. Open PowerShell Core 7 Preview-4 in Windows Terminal
1. Left click select multiple lines, and then right click.

# Expected behavior

Right click should copy text.

# Actual behavior

Crash. Occurs 100% of the time following these steps. I've tried it 6 times and got the crash all 6  times.

I believe this is a regression as this did not occur in the previous version of Windows Terminal Preview.

**Comment by DHowett-MSFT** [comment]
Sorry! I absolutely missed your existing feedback link.

**Comment by ZeddG93** [comment]
I use shortcuts `Ctrl+C` to copy text under WSL, and occasional crashes occur.

**Comment by drichardson** [comment]
Quick test confirms this is fixed in v0.5.2762.0.
